setup.py

```diff
@@ -21,6 +21,7 @@
         'beautifulsoup4>=4.4',
         'lxml>=3.6',
         'pandas>=0.18',
+        'pytest>3.1',
```

Comment:
I completely agree. Thank you for the feedback.

I will make this improvement.